### PR TITLE
chore: min/max logic.

### PR DESCRIPTION
**Pull request by rcresnik**

### CODE REVIEW

File: brd-ios/breadwallet/src/Redesign/Scenes/Swap/Swap/SwapInteractor.swift [hunk 1]
```diff
@@ -9,8 +9,23 @@
...
+    var errorMessage: String {
+        switch self {
+        case .noQuote:
+            return "No quote for currency pair."
+            
+        case .general:
+            return "Something went wrong"
+            
+        case .toSmall:
+            return "Entered crypto amount was to small. Has to be above 50 usd"
```
Comment (kenan-mamedoff) [inline brd-ios/breadwallet/src/Redesign/Scenes/Swap/Swap/SwapInteractor.swift:26]: too* USD*

if this is temporary, then ignore.

File: brd-ios/breadwallet/src/Redesign/Scenes/Swap/Swap/SwapInteractor.swift [hunk 2]
```diff
@@ -182,16 +197,21 @@ class SwapInteractor: NSObject, Interactor, SwapViewActions {
...
     
     func setAmount(viewAction: SwapModels.Amounts.ViewAction) {
+        var viewAction = viewAction
         dataStore?.minMaxToggleValue = viewAction.minMaxToggleValue
         
         if let minMaxToggleValue = viewAction.minMaxToggleValue {
             switch minMaxToggleValue {
             case .min:
-                dataStore?.fromFiatAmount = 50
+                viewAction.fromFiatAmount = "50"
```
Comment (kenan-mamedoff) [inline brd-ios/breadwallet/src/Redesign/Scenes/Swap/Swap/SwapInteractor.swift:206]: 50 should come from somewhere. is it mocked for now?

File: brd-ios/breadwallet/src/Redesign/Scenes/Swap/Swap/SwapInteractor.swift [hunk 3]
```diff
@@ -182,16 +197,21 @@ class SwapInteractor: NSObject, Interactor, SwapViewActions {
...
                 
             case .max:
-                dataStore?.fromCryptoAmount = getBaseBalance()?.tokenValue
-                
+                guard (getBaseBalance()?.fiatValue ?? 0) >= 50 else {
```
Comment (kenan-mamedoff) [inline brd-ios/breadwallet/src/Redesign/Scenes/Swap/Swap/SwapInteractor.swift:209]: same

**Review by dangelovska — APPROVED** [review]

File: brd-ios/breadwallet/src/Redesign/Scenes/Swap/Swap/SwapInteractor.swift (hunk 1 above)
Comment (rcresnik) [reply brd-ios/breadwallet/src/Redesign/Scenes/Swap/Swap/SwapInteractor.swift:26]: this needs 2 be localized anyhow.. i just added them cause its way easier 2 debug the app this way

File: brd-ios/breadwallet/src/Redesign/Scenes/Swap/Swap/SwapInteractor.swift (hunk 2 above)
Comment (rcresnik) [reply brd-ios/breadwallet/src/Redesign/Scenes/Swap/Swap/SwapInteractor.swift:206]: i only updated the logic.. hardcoded 50 was there before.. should prolly be a constant somewhere

**Pull request merged**